scripts/generate_querygraph.py

```diff
@@ -0,0 +1,45 @@
+# generate_querygraph.py
```

Comment:
Let's include this code and ship it as an executable module instead

Reply:
So that means it runs as `python3 -m generate_querygraph explain_output.json` ?

Reply:
Like

```sh
python -m duckdb.query_graph ...
```

But basically yeah. Could even publish it an standalone executable on the user machines that way too